property_table.py

```diff
@@ -354,44 +354,45 @@ class PropertyTable:
             False,
             ["PartType5/SubgridMasses", "PartType5/Velocities"],
         ),
-        "BaryonInertiaTensor": (
-            "BaryonInertiaTensor",
+        "DarkMatterInertiaTensor": (
```

Comment:
Will people clearly know that the "Inertia Tensor" is not the usual (non-iterative) "Inertia Tensor" computed by them?

Reply:
I think the description makes this clear, so I'm inclined to leave the name as it is